doc/docs/master/reference/scaling_changes.md

```diff
@@ -0,0 +1,70 @@
+# Pipeline Scaling Limitation in Community Edition
+
+!!! Warning "We are here to help!" 
+    Scaling Pachyderm pipelines is easy to do but hard for users to perfect. 
+
+It can be difficult to balance the breaking down of data science systems into multiple pipelines, horizontally scaling those pipelines to optimize the processing time and effectively understanding Pachyderm's unique data-first approach to development in connecting those pipelines. Lalalal
+
+Starting with this new 1.13.0 release, we introduce a **limit to the number of pipelines and their parallelization** Community Edition users can deploy.
+
+These limits have been set high enough for most of our users to benefit from Pachyderm Community Edition capabilities while allowing us to **better assist in designing, scaling, and running pipelines** those whose workload and pipeline numbers require finer tuning to handle their workload efficiently.
+
+## Scaling Limitation
+
+Our new limitation have been set on:
+
+- The **number of pipelines** deployed: Community Users can deploy **up to 16 pipelines**.
+- The **number of workers** for each pipeline: Community Users can run **up to 8 workers in parallel** on each pipeline.
+
+Our Customer Success Experience has shown us that hitting those limits means that you are likely to be needing our assistance in optimizing your DAGs.
+
+## Lift the limitation
+
+We provide an easy way to request a **2 weeks Enterprise Token** that will help you experiment with Pachyderm without limit. //TODO insert a link to the Request token page here
```

Comment:
"2 weeks Enterprise Token" is not proper English.

Reply:
I'm also not sure you should mention the precise length of the token.  If we choose to change it, or A/B test different lengths, you'll be changing this document often.

Reply:
Removing every mention of time for the free trial of the Enterprise Edition.